docs/design-notes/wasm.md

```diff
@@ -112,42 +110,42 @@ for memory management.
 Example source code:
 
 ```c
-struct __attribute__((packed)) nullable_bigint { 
-    int size;
-    long long v;
-};
+const int WASM_PAGE_SIZE = 64 * 1024;
+
+const int _scylla_abi = 0;
```

Comment:
Does this export a 0, or an address of a zero?

btw, we should start with 1, 0 is too easy to get by accident.

Reply:
It exports the address or the zero. It was a little problematic, because it is different in AssemblyScript - the actual value gets exported. For now I changed the example for it to export an array with a 0 and the generated wasm looked ok, but I haven't worked on AssemblyScript yet so it may not be the best solution 
I don't have anything against indexing from 1, so I'll change it in the next version.